ui/src/components/Preview/Overview/MetricContent/MetricContent.tsx

```diff
@@ -0,0 +1,78 @@
+import { DashboardMetricOverview } from "../../../../api/Api";
+import { themeColors } from "../../../../utils/staticData";
+import PieChart from "../../../UI/Piechart";
+import MetricInfoIcon from "../MetricInfoIcon";
+import styles from "./MetricContent.module.css";
+
+interface IMetricContentProps {
```

Comment:
This type can remain here or be moved to a separate file!  
:: If you want this type [Props] to be here, keep it the same for every file.

Reply:
Thank You, I will keep props in every file.